spec/models/resource_action_spec.rb

```diff
@@ -62,8 +62,7 @@
       it "validates queue entry" do
         targets = [FactoryGirl.create(:vm_vmware), FactoryGirl.create(:vm_vmware)]
         ae_attributes[:target_object_type] = targets.first.class.base_class.name
-        klass = targets.first.id.class
-        ae_attributes['Array::target_object_ids'] = targets.collect { |t| "#{klass}::#{t.id}" }.join(",")
+        ae_attributes['Array::target_object_ids'] = targets.collect { |t| "#{t.class.name.demodulize}::#{t.id}" }.join(",")
```

Comment:
@d-m-u 
Is there a way to test and see if its broken  by doing a multi select and invoke a custom button on the, does Automate complain about invalid objects?

Reply:
There's a bug open about this very thing so I think that kinda proves it's currently broken.

Reply:
It would mean that this feature has not been QE'ed if its that broken.

Reply:
Since QE is the one who opened this bug, that sounds about right.